src/devices/Bmxx80/README.md

```diff
@@ -17,7 +17,7 @@ The implementation supports the following devices:
 
 3 examples on how to use this device binding are available in the [samples](samples) folder.
 
-The following fritzing diagram illustrates one way to wire up the BMP280 with a Raspberry Pi using I2C:
+The following fritzing diagram illustrates one way to wire up the BMP280 with a Raspberry Pi using I²C:
```

Comment:
FYI: I'm fine with this in the docs but will push back on similar changes to `.cs`

Reply:
Had missed that while reverting the squared symbol. Thanks.